src/canvas/dotcanvas.jl

```diff
@@ -9,17 +9,13 @@ DOT_DECODE[4] = ':'
 DOT_DECODE[(N_DOT + 1):typemax(UInt16)] = UNICODE_TABLE[1:(typemax(UInt16) - N_DOT)]
 
 """
-Similar to the `AsciiCanvas`, the `DotCanvas` only uses
-ASCII characters to draw it's content. Naturally,
-it doesn't look quite as nice as the Unicode-based
-ones. However, in some situations it might yield
-better results. Printing plots to a file is one
-of those situations.
+Similar to the `AsciiCanvas`, the `DotCanvas` only uses ASCII characters to draw it's content.
+Naturally, it doesn't look quite as nice as the Unicode-based ones.
+However, in some situations it might yield better results.
+Printing plots to a file is one of those situations.
```

Comment:
Just dropping a note to commend you for using semantic line breaks instead of a fixed width :)

Reply:
Thanks, I didn't know that it was called that way, it just felt more readable ;)

Reply:
It's actually a [best practice](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) that goes back all the way to the early Unix days :)

Reply:
1974 ! Wasn't even born, dully noted :+1: